addons/openebs/2.6.0/install.sh

```diff
@@ -0,0 +1,265 @@
+    fi
+    if [ -z "$OPENEBS_CSTOR_STORAGE_CLASS" ]; then
+        OPENEBS_CSTOR_STORAGE_CLASS=openebs-cstor
+    fi
+    if [ -z "$OPENEBS_CSTOR_TARGET_REPLICATION" ]; then
+        OPENEBS_CSTOR_TARGET_REPLICATION="3"
+    fi
+}
+
+function openebs() {
+    local src="$DIR/addons/openebs/$OPENEBS_VERSION"
+    local dst="$DIR/kustomize/openebs"
+
+    render_yaml_file "$src/tmpl-kustomization.yaml" > "$dst/kustomization.yaml"
+    render_yaml_file "$src/tmpl-namespace.yaml" > "$dst/namespace.yaml"
+    cp "$src/operator.yaml" "$dst/"
+
+    openebs_upgrade
+
+    if [ "$OPENEBS_LOCALPV" = "1" ]; then
+        render_yaml_file "$src/tmpl-localpv-storage-class.yaml" > "$dst/localpv-storage-class.yaml"
+        insert_resources "$dst/kustomization.yaml" localpv-storage-class.yaml
+
+        if [ "$OPENEBS_LOCALPV_STORAGE_CLASS" = "default" ]; then
+            render_yaml_file "$src/tmpl-patch-localpv-default.yaml" > "$dst/patch-localpv-default.yaml"
+            insert_patches_strategic_merge "$dst/kustomization.yaml" patch-localpv-default.yaml
+        fi
+
+        kubectl apply -k "$dst/"
+    fi
+
+    if [ "$OPENEBS_CSTOR" = "1" ]; then
+        openebs_iscsi
+
+        cp "$src/cstor-provisioner.yaml" "$dst/"
+        insert_resources "$dst/kustomization.yaml" cstor-provisioner.yaml
+
+        kubectl apply -k "$dst/"
+
+        echo "Waiting for OpenEBS operator to apply CustomResourceDefinitions"
+        spinner_until 120 kubernetes_resource_exists default crd storagepoolclaims.openebs.io
+
+        dst="$dst/storage"
+        mkdir -p "$dst"
+        touch "$dst/kustomization.yaml"
+
+        # create the storage pool claim
+        openebs_cstor_max_pools
+        render_yaml_file "$src/tmpl-storage-pool-claim.yaml" > "$dst/storage-pool-claim.yaml"
+        insert_resources "$dst/kustomization.yaml" storage-pool-claim.yaml
+
+        # create the storage class
+        OPENEBS_CSTOR_REPLICA_COUNT="$OPENEBS_CSTOR_MAX_POOLS"
+        if [ $OPENEBS_CSTOR_REPLICA_COUNT -gt 3 ]; then
+            OPENEBS_CSTOR_REPLICA_COUNT=3
+        fi
+        render_yaml_file "$src/tmpl-cstor-storage-class.yaml" > "$dst/cstor-storage-class.yaml"
+        insert_resources "$dst/kustomization.yaml" cstor-storage-class.yaml
+        if [ "$OPENEBS_CSTOR_STORAGE_CLASS" = "default" ]; then
+            render_yaml_file "$src/tmpl-patch-cstor-default.yaml" > "$dst/patch-cstor-default.yaml"
+            insert_patches_strategic_merge "$dst/kustomization.yaml" patch-cstor-default.yaml
+        fi
+
+        kubectl apply -k "$dst/"
+
+        if ! openebs_cstor_pool_count "$OPENEBS_CSTOR_MAX_POOLS"; then
+            if [ "$OPENEBS_CSTOR_MAX_POOLS" = "1" ]; then
+                printf "${GREEN}Waiting for 1 disk${NC}\n"
+            else
+                printf "${GREEN}Waiting for ${OPENEBS_CSTOR_MAX_POOLS} disks${NC}\n"
+            fi
+            spinner_until 300 openebs_cstor_pool_count "$OPENEBS_CSTOR_MAX_POOLS"
```

Comment:
Don't we usually wait for someone to attach the disk in Rook? Not opposed to this just a behavior change.

Reply:
Yeah, but this is what we've done so far with openebs

Reply:
Wasn't this indefinitely waiting though: https://github.com/replicatedhq/kURL/blob/f64d562cdb582974f1967bf9eaa6e8cc53377721/addons/openebs/1.12.0/install.sh#L86

Again, I wouldn't let this hold up me approving the PR.

Reply:
oh, that's what you meant
this just makes it so that if nothing attaches the disk within 5 minutes it's a failure

I can move it back to 'forever' if you want

Reply:
NM, It's cool. In the words of Big Krit `4eva Is a Mighty Long Time`.